src/Compilers/Core/Portable/Syntax/SyntaxList`1.cs

```diff
@@ -395,7 +395,7 @@ private TNode[] Nodes
         /// </summary>
         public Enumerator GetEnumerator()
         {
-            return new Enumerator(this);
+            return new Enumerator(in this);
```

Comment:
❓ Why make this change? The current structure appears to only have one field, which is `SyntaxNode` (a reference type) and therefore be cheapest to copy by value.

Reply:
Will take a look. I may have mixed it up with other cases. There would not be much point to pass a single reference by `in`.